backtrace-library/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
     package="backtraceio.library">
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
why we need camera permission?

Reply:
I think we might need camera permission to handle the system broadcast intent:
`android.intent.action.CAMERA_BUTTON`

But I haven't been able to test it yet

Reply:
Does it mean that our users also need to add camera permission? With camera permission in the application (not in library) can we handle camera events?

Reply:
I'm actually not sure about that @konraddysput, I just assumed we would have to add it in both places since we already add `EXTERNAL_STORAGE` and `INTERNET` permissions in both places.

Reply:
we need `EXTERNAL_STORAGE` and `INTERNET` because library is storing files and sending it via Internet. I think that our library should not require camera permissions. Can you test without this permission?

Reply:
I will test without the permission and remove if it's unnecessary

Reply:
I don't even think most android devices have a hardware camera button, and we would detect this event anyway since our app would be backgrounded if the camera is opened. We can remove this permission and not worry about `android.intent.action.CAMERA_BUTTON` for now.